core/story-api/src/main/java/org/lgna/story/Key.java

```diff
@@ -252,12 +252,11 @@ private Key(int keyCode) {
     MapContainer.map.put(this.keyCode, this);
   }
 
```

Comment:
this one is safe, because it is an int, right?

Reply:
Yeah, if the int value changes there is something bigger to worry about.